tests/integration/skipping/skipping_test.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+from typing import List
+
+import pytest
+
+from protostar.commands.test.test_result_formatter import format_test_result
+from protostar.commands.test.test_results import SkippedTestCaseResult
+
+from tests.integration.conftest import (
+    RunCairoTestRunnerFixture,
+    assert_cairo_test_cases,
+)
+
+
+@pytest.mark.asyncio
+async def test_testing_output(
+    run_cairo_test_runner: RunCairoTestRunnerFixture,
+):
+    testing_summary = await run_cairo_test_runner(
+        Path(__file__).parent / "skipping_test.cairo"
+    )
+
+    assert_cairo_test_cases(
+        testing_summary,
+        expected_failed_test_cases_names=["test_skip_outside_of_setup"],
+        expected_skipped_test_cases_names=[
+            "test_skip",
+            "test_skip_no_reason",
+        ],
+    )
+
+    output = get_formatted_output(testing_summary.skipped)
```

Comment:
We do not test formatting in integration tests, but rather by unit testing TestingSummary, or by doing E2E tests.

This test should rather assert that expected data structures are returned from test runner.

Reply:
Added E2E test